packages/server/src/app/db/users/users.service.ts

```diff
@@ -185,6 +186,21 @@ export class UsersService {
         await this.userModel.updateOne({ "_id": userId }, { "stats.blogs": blogCount });
     }
 
+    /**
+     * Updates the number of blogs a user has tied to their account. Should only count blogs
+     * that are published and not deleted.
+     * 
+     * @param user The user to update
+     * @param addOne Whether to add one or subtract one
+     */
+    async changeBlogCount(user: JwtPayload, addOne: boolean): Promise<void> {
```

Comment:
The doc and name make me think this sets the count arbitrarily, but it's really just incrementing or decrementing. The name/commentdoc should probably be changed to reflect that.